Sources/Swiftly/Install.swift

```diff
@@ -40,9 +40,15 @@ struct Install: SwiftlyCommand {
             $ swiftly install 5.7-snapshot
             $ swiftly install main-snapshot
 
-         Install whatever toolchain is currently selected, such as the the one in the .swift-version file:
+        Install whatever toolchain is currently selected, such as the the one in the .swift-version file:
 
             $ swiftly install
+
+        NOTE: Toolchains are downloaded to a temporary file that is later cleaned up by the installation \
```

Comment:
```suggestion
        NOTE: Swiftly downloads toolchains to a temporary file that it later cleans during its installation process. If these files are too big for your system temporary directory, set another location by setting the `TMPDIR` environment variable.
```

(since this one spans multiple lines below, I shoved it all into a single line above - so if you take this tweak, remove lines 48 and 49 as redundant.